cmd/aries-agentd/main_test.go

```diff
@@ -210,3 +216,16 @@ func TestStartAriesWithoutInboundHost(t *testing.T) {
 	}
 
 }
+
+func setupLevelDB(t testing.TB) (string, func()) {
```

Comment:
This is only creating a temporary directory (with cleanup) - it is not setting up leveldb

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/79f650938572126d2758278779bf911ef7c090a4..6f3a7d7e19a26a670b5002e34a70ea25c315587e#diff-7d0c01660ad47362e88877a74666c805R220).